opt-tools/docker/images/app/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM openjdk:11-jre-slim
+CMD cd ../../../../; mvn package;
+COPY target/docker-practice-java-0.0.1-SNAPSHOT.jar /usr/local/lib/docker-practice-java-0.0.1-SNAPSHOT.jar
```

Comment:
Copying jar-artifact into `/usr/local/lib/` path is not good practice. Please do not use system paths for such things. You could use `/opt` directory.

Reply:
I will also take into account and correct it.